src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -146,18 +151,13 @@ class BaseInvertedFlatList extends Component {
 
                 // Native platforms do not need to measure items and work fine without this.
                 // Web requires that items be measured or else crazy things happen when scrolling.
-                // eslint-disable-next-line react/jsx-props-no-multi-spaces
                 getItemLayout={this.props.shouldMeasureItems ? this.getItemLayout : undefined}
                 bounces={false}
 
                 // We keep this property very low so that chat switching remains fast
-                // eslint-disable-next-line react/jsx-props-no-multi-spaces
                 maxToRenderPerBatch={1}
                 windowSize={15}
-
-                // Setting removeClippedSubviews will break text selection on Android
-                // eslint-disable-next-line react/jsx-props-no-multi-spaces
-                removeClippedSubviews={false}
```

Comment:
@AndrewGable do you remember how this broke text selection on Android? I tested it and seemed to work ok to me, but maybe I misinterpreted this comment.

Reply:
I believe it was something along the lines of https://stackoverflow.com/a/66703331/2509962

Reply:
Chatted with Andrew and neither one of us could remember why it was added. We're also going to implement a long press menu to select the entire contents of a message IIRC so whatever this refers to (still not entirely sure) might be even less relevant in the future.